fdbclient/SystemData.cpp

```diff
@@ -641,13 +641,22 @@ const KeyRangeRef restoreApplierKeys(LiteralStringRef("\xff\x02/restoreApplier/"
 const KeyRef restoreApplierTxnValue = LiteralStringRef("1");
 
 // restoreApplierKeys: track atomic transaction progress to ensure applying atomicOp exactly once
+// Version integer must be BigEndian to maintain ordering in lexical order
 const Key restoreApplierKeyFor(UID const& applierID, Version version) {
 	BinaryWriter wr(Unversioned());
-	wr.serializeBytes(restoreWorkersKeys.begin);
+	wr.serializeBytes(restoreApplierKeys.begin);
```

Comment:
put the bigEndian calls into this function and the decoder to prevent future people from making the same mistake